### PR TITLE
convert word to markdown of charter

### PR DESCRIPTION
**Pull request by christo4ferris**
converted word doc to markdown for further maintenance. admittedly, this is as much html as markdown, but the nested ordered lists necessitated this approach.

Signed-off-by: Chris Ferris <chrisfer@us.ibm.com>

### CODE REVIEW

**Comment by dlorenc** [comment]
> converted word doc to markdown for further maintenance. admittedly, this is as much html as markdown, but the nested ordered lists necessitated this approach.

Which is meant to be the "source of truth" going forward? If it's this one, could we also delete the PDF copy to prevent them from getting out of sync?

File: openssf-charter.md [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+        <li>approve a budget directing the use of funds raised by the OpenSSF from all sources of revenue for the OpenSSF;</li>
+              <br/>
+        <li>approve directed fundraising proposals for specific Working Groups or Projects that will raise and spend funds within the Working Group or Project;</li>
+              <br/>
+        <li>establish any conformance programs and solicit input
+            > (including testing tools) from the applicable governance
+            > body of any Technical Initiative for defining and
+            > administering any programs related to conformance with any
+            > Technical Initiative (each, a "Conformance Program");</li>
+              <br/>
```
Comment (dcmiddle) [inline openssf-charter.md:277]: remove ">"s

File: openssf-charter.md [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+<br/>
+  <ol type="i">
+    <li>ten OpenSSF Member Representatives selected by the OpenSSF Members; </li>
+<br/>
+    <li>one TAC Representative (as defined herein);</li>
+<br/>
+    <li>one Associate Member Representative appointed by the OSSC
+             Governing Board; and</li>
+<br/>
+    <li>iv. one Security Community Individual Representative elected by contributors to Technical Initiatives.</li>
```
Comment (dcmiddle) [inline openssf-charter.md:201]: remove "iv."

File: openssf-charter.md [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+        </br>
+  <li>In the event an unbudgeted or otherwise unfunded obligation arises related to the OpenSSF, the Linux Foundation will coordinate with the Governing Board to address gap funding requirements.</li></ol>
+
+13. **General & Administrative Expenses**
+<ol type="a">
+  <li>The Linux Foundation will have custody of and final authority over the usage of any fees, funds and other cash receipts.</li>
+        </br>
+  <li>A General & Administrative (G&A) fee will be applied by the Linux Foundation to funds raised to cover membership records, finance, accounting, and human resources operations. The G&A fee will be 9% of the OpenSSF's first \$1,000,000 of gross receipts each year and 6% of the OpenSSF's gross receipts each year over \$1,000,000. Individual Technical Initiative funding arrangements may be setup under alternative arrangements by approval of the Governing Board and the Linux Foundation.</li></ol>
```
Comment (dcmiddle) [inline openssf-charter.md:410]: For some reason these "\"s render. See if you can get by without the escapes in front of the $s.

**Review by dcmiddle — CHANGES_REQUESTED** [review]
This will be great to have this diff'able from now on.
I only see some minor formatting defects in the conversion.

**Comment by christo4ferris** [comment]
> > converted word doc to markdown for further maintenance. admittedly, this is as much html as markdown, but the nested ordered lists necessitated this approach.
> 
> Which is meant to be the "source of truth" going forward? If it's this one, could we also delete the PDF copy to prevent them from getting out of sync?

I was thinking the markdown would be source of truth

File: openssf-charter.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (christo4ferris) [reply openssf-charter.md:201]: thanks

File: openssf-charter.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (christo4ferris) [reply openssf-charter.md:410]: hmmm... seems VSCode markdown preview inconsistent with GH

**Comment by christo4ferris** [comment]
We discussed on the Planning Committee call. Apparently there is a formal process of managing the charter as a Word doc per Lindsay. Closing this out given the conflict as it would be a challenge to keep the source of truth consistent. with a markdown version due to the complexity of the markup.

**Pull request closed without merging**